src/__init__.py

```diff
@@ -53,6 +53,7 @@ def read_config(filename=None, options=None):
             'use_replication_slots': 'no',
             'max_rewind_retries': 3,
             'postgres_timeout': 60,
+            'election_loser_timeout': 0,  # Timeout for election losers. For test purposes only.
```

Comment:
let's move this  optio to debug section and make more general.
Something like
```
debug:
   delay_name: 'election_looser',
   delay_value: 30,
```

by analogy with
```
if self.config.get('debug', 'failure_name', fallback=None) == name:
```